### ISSUE TITLE
Generating training data from SUNCG

### ISSUE THREAD

**Issue by mgarbade** [shurans/sscnet#35]
On my computer, the generation of training data takes ~9min per scene. With 45k scenes this would take me ~ 280 days on one machine...  
Am I doing something wrong?  
Should I simply download the precomputed image/depth pairs from the paper ["Physically-Based Rendering for Indoor Scene Understanding"](http://3dvision.princeton.edu/projects/2016/PBRS/). If yes, is their image retrieving process the same, so the training is  equivalent to `sscnet`?  

**Edit:**
I just realized that "Physically-Based Rendering for Indoor Scene Understanding" does not provide ground truth 3D scene labels, which would be needed for the sscnet-training, so I guess I'm back to extracting the dataset `genSUNCGdataScript(sceneId)`. Any idea how to speed up the process, would be appreciated...

**Edit2:**
Where can I find the train / test / val splits?

**Comment by KnightOfTheMoonlight** [comment]
You can just ask shuran for suncg dataset. 
From her project homepage [sscnet](http://sscnet.cs.princeton.edu/), there's the illustration for dataset access.

> If you are a researcher and would like to get access to the data, please print and sign this [agreement] and email it to suncgteam@googlegroups.com.
